pom.xml

```diff
@@ -36,63 +41,63 @@
 </dependencies>
 
 <!-- Build! -->
-<build>
-    <plugins>
-        <!-- Build -> plugin -> surefire -->
-        <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-surefire-plugin</artifactId>
-            <version>3.0.0-M5</version>
-        </plugin>
-        <!-- Build -> plugin -> jacoco -->
-        <plugin>
-            <groupId>org.jacoco</groupId>
-            <artifactId>jacoco-maven-plugin</artifactId>
-            <version>0.8.6</version>
-            <executions>
-                <execution>
-                    <id>coverage-initialize</id>
-                    <goals>
-                        <goal>prepare-agent</goal>
-                    </goals>
-                </execution>
-                <execution>
-                    <id>coverage-report</id>
-                    <phase>post-integration-test</phase>
-                    <goals>
-                        <goal>report</goal>
-                    </goals>
-                </execution>
-            </executions>
-        </plugin>
+<!-- <build> -->
+<!--     <plugins> -->
+<!--         <!-1- Build -> plugin -> surefire -1-> -->
+<!--         <plugin> -->
+<!--             <groupId>org.apache.maven.plugins</groupId> -->
+<!--             <artifactId>maven-surefire-plugin</artifactId> -->
+<!--             <version>3.0.0-M5</version> -->
+<!--         </plugin> -->
+<!--         <!-1- Build -> plugin -> jacoco -1-> -->
+<!--         <plugin> -->
+<!--             <groupId>org.jacoco</groupId> -->
+<!--             <artifactId>jacoco-maven-plugin</artifactId> -->
+<!--             <version>0.8.6</version> -->
+<!--             <executions> -->
+<!--                 <execution> -->
+<!--                     <id>coverage-initialize</id> -->
+<!--                     <goals> -->
+<!--                         <goal>prepare-agent</goal> -->
+<!--                     </goals> -->
+<!--                 </execution> -->
+<!--                 <execution> -->
+<!--                     <id>coverage-report</id> -->
+<!--                     <phase>post-integration-test</phase> -->
+<!--                     <goals> -->
+<!--                         <goal>report</goal> -->
+<!--                     </goals> -->
+<!--                 </execution> -->
+<!--             </executions> -->
+<!--         </plugin> -->
```

Comment:
Why are we disabling the build step? If it's no longer needed just remove the code instead of having zombie code here. We can always go back to the commit before and grab the code if we'll need it again.